File: warmups/week11/day03_isogram_finder/isogram.js
const isIsogram = (word) => {
  // let newWord = new Set(word.toLowerCase());
  //
  // if (newWord.size === word.length) {
  //   return true;
  // } else {
  //   return false;
  // }
  return new Set(word.toLowerCase()).size === word.length;
}


console.log(isIsogram('Dermatoglyphics')); //true
console.log(isIsogram('isogram'));  //true
console.log(isIsogram('aba')); //false
console.log(isIsogram('moOse')); //false
console.log(isIsogram('isIsogram')); //false
console.log(isIsogram('')); //true

File: warmups/week11/day03_isogram_finder/isogram.rb
def isogram word
  word.length == word.downcase.split('').uniq.length
end

p isogram 'Dermatoglyphics'  #true
p isogram 'isogram'   #true
p isogram 'aba'    #false
p isogram 'moOse'    #false
p isogram 'isIsogram'    #false
p isogram ''     #true
